engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -836,11 +1539,11 @@ public LogicalPlanConfiguration addFromJson(JSONObject json, Configuration conf)
     JSONArray operatorArray = json.getJSONArray("operators");
```

Comment:
This method is also rather dense, can it be refactored a bit to make it easier to understand?